bc_obps/registration/schema/v2/operation.py

```diff
@@ -1,9 +1,23 @@
 from typing import List, Optional, Union
+from registration.schema.v1.contact import ContactIn
 from ninja import Field, FilterSchema, ModelSchema, Schema
 from registration.models import Operation
+from pydantic import ValidationError, model_validator
+from ninja.types import DictStrAny
 
 #### Operation schemas
 
+class OperationRepresentativeIn(Schema):
+    operation_representatives: Optional[List[int]] = []
+    new_operation_representatives: Optional[List[ContactIn]] = []
+
+    @model_validator(mode='after')
```

Comment:
Struggling with mypy here. When I check the type of data I get:
```
(Pdb) type(data)
<class 'registration.schema.v2.operation.OperationRepresentativeIn'>
```
But when I give it OperationRepresentativeIn it is unhappy

Reply:
It sounds like mypy may not play nice with after validators: https://github.com/pydantic/pydantic/issues/6277